perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -1759,5 +1758,72 @@ public Member call(ApiCaller ac, Deserializer parms) throws PerunException {
 					ac.getGroupById(parms.readInt("group")),
 					status);
 		}
+	},
+
+	/*#
+	 * Returns all groups which can be registered into during vo registration.
+	 *
+	 * @throw VoNotExistsException When the vo doesn't exist
+	 *
+	 * @param vo int vo <code>id</code>
+	 * @return List<Group> list of groups
+	 */
+	getGroupsToAutoRegistration {
+		@Override
+		public List<Group> call(ApiCaller ac, Deserializer parms) throws PerunException {
+			return ac.getGroupsManager().getGroupsForAutoRegistration(ac.getSession(),
+				ac.getVoById(parms.readInt("vo")));
+		}
+	},
+
+	/*#
+	 * Deletes groups from a list of groups which can be registered into during vo registration.
+	 *
+	 * @throw GroupNotExistsException When the group doesn't exist
+	 * @throw GroupAlreadyRemovedException When deleting group which is already removed
+	 *
+	 * @param ids List<Integer> list of groups IDs
+	 */
+	deleteGroupsToAutoRegistration {
```

Comment:
I think this is a typo.
```suggestion
	deleteGroupsFromAutoRegistration {
```

Reply:
Fixed, thanks.